docs/modules/user/pages/quickstart.adoc

```diff
@@ -364,6 +365,193 @@ docker run \
+                                                                                         
++-----------------------------------------------------------------------------------------+
+| Processes:                                                                              |
+|  GPU   GI   CI        PID   Type   Process name                              GPU Memory |
+|        ID   ID                                                               Usage      |
+|=========================================================================================|
+|   No running processes found                                                            |
++-----------------------------------------------------------------------------------------+
+```
+
+`ls -l /dev/nvidia*`
+```
+crw-rw-rw- 1 root root 195,   0 Aug 15 10:44 /dev/nvidia0
+crw-rw-rw- 1 root root 195, 255 Aug 15 10:44 /dev/nvidiactl
+crw-rw-rw- 1 root root 195, 254 Aug 15 10:44 /dev/nvidia-modeset
+crw-rw-rw- 1 root root 506,   0 Aug 15 10:44 /dev/nvidia-uvm
+crw-rw-rw- 1 root root 506,   1 Aug 15 10:44 /dev/nvidia-uvm-tools
+
+/dev/nvidia-caps:
+total 0
+cr-------- 1 root root 509, 1 Aug 15 10:44 nvidia-cap1
+cr--r--r-- 1 root root 509, 2 Aug 15 10:44 nvidia-cap2
+```
+
+.I am not able to see all the Nvidia devices
+[%collapsible]
+====
+You may need to setup your host to automatically load the NVIDIA GPU Kernel modules at boot time.
+
+First, create a new file `nvidia.conf` in the `/etc/modules-load.d/` directory and open it with a text editor.
+
+`nano /etc/modules-load.d/nvidia.conf`
+
+paste the following content to the file:
+```
+nvidia
+nvidia_uvm
+```
+
+For the changes to take effect, update the `initramfs` file with the following command:
+`update-initramfs -u`
+
+Add udev rules to add missing Nvidia devices in PVE 8
+`nano /etc/udev/rules.d/70-nvidia.rules`
+paste the following content to the file:
+
+```
+# create necessary NVIDIA device files in /dev/*
+KERNEL=="nvidia", RUN+="/bin/bash -c '/usr/bin/nvidia-smi -L && /bin/chmod 0666 /dev/nvidia*'"
+KERNEL=="nvidia_uvm", RUN+="/bin/bash -c '/usr/bin/nvidia-modprobe -c0 -u && /bin/chmod 0666 /dev/nvidia-uvm*'"
+```
+
+`reboot` you PVE host and try running `ls -l /dev/nvidia*` again.
+====
+
+
+Add the udev rules to the host as explained in the xref:quickstart.adoc#_virtual_devices_support[Virtual devices support] section.
+
+One last final check on the host: we have to make sure that the `nvidia-drm` module has been loaded and that the module is loaded with the flag `modeset=1`.
+
+[source,bash]
+....
+cat /sys/module/nvidia_drm/parameters/modeset
+Y
+....
+
+.I get `N` or the file is not present, how do I set the flag?
+[%collapsible]
+====
+
+The easiest way to make the change persistent is to add `nvidia-drm.modeset=1` to the `GRUB_CMDLINE_LINUX_DEFAULT` line in `/etc/default/grub` ex:
+
+....
+GRUB_CMDLINE_LINUX_DEFAULT="quiet nvidia-drm.modeset=1"
+....
+
+Then `sudo update-grub` and `reboot`.
+====
```

Comment:
I'm sorry to be a pain in the ass (and I'd be happy to do the edit myself if you don't want to) but if we remove this section wouldn't this tab be valid for all LXC setups no matter which GPU they've got?  

The instructions below are pretty good to get a LXC container up and running, then we just say to complete the installation based on the GPU following the other tabs. 

The udev rules and the kernel load stuff should probably be added to the Nvidia tab as well, they might get handy if some OS doesn't include those.

Reply:
hahahaha no worries. I see what you mean, it makes sense.

I will remove the top part and just add a small note saying: You should make sure you have the proper GPU drivers and input udev rules on your PVE host before proceeding and will move the nvidia udev rules into the nvidia tabs.